tools/src/versioning/android/expoModulesTransforms.ts

```diff
@@ -0,0 +1,76 @@
+import escapeRegExp from 'lodash/escapeRegExp';
+
+import { FileTransforms } from '../../Transforms.types';
+import { packagesToKeep, packagesToRename } from './packagesConfig';
+import { deleteLinesBetweenTags } from './utils';
+
+function expoModulesBaseTransforms(abiVersion: string): FileTransforms {
+  return {
+    path: [
+      {
+        find: /src\/main\/java/,
```

Comment:
`find: string` should work here as far as I remember 🤔 It would look a bit better without having to escape slashes 😉
(The same applies to other rules below)